File: diKoding/DotPeekServer/DotPeekServerCs/BlogContext.cs
namespace DotPeekServer
{
    using System.Data.Entity;

    public partial class BlogContext 
        : DbContext
    {
        public BlogContext()
            : base(@"name=BlogContext")
        {
        }

        public DbSet<Blog> Blogs { get; set; }

        protected override void OnModelCreating(
            DbModelBuilder modelBuilder)
        {
        }
    }
}

File: diKoding/DotPeekServer/DotPeekServerVb/BlogContext.vb
Imports System.Data.Entity

Partial Public Class BlogContext
    Inherits DbContext

    Public Sub New()
        MyBase.New("name=BlogContext")
    End Sub

    Public Property Blogs As DbSet(Of Blog)

    Protected Overrides Sub OnModelCreating(modelBuilder As DbModelBuilder)
    End Sub
End Class